### PR TITLE
Improve README and add installation problems section

### PR DESCRIPTION
**Pull request by solidgoldpig**

### CODE REVIEW

File: installation.js [hunk 1]
```diff
@@ -2,12 +2,23 @@ const {app} = require('./app.js')
...
 
 const ipc = require('electron-better-ipc')
 
 const installEditorDependencies = (reinstall) => {
   app.notify(`${reinstall ? 'Reinstalling' : 'Installing'} editor dependencies`)
-  execSync(`. ${app.paths.nvs}/nvs.sh && nvs add 10.11 && nvs use 10.11 && cd ${app.paths.editor} && npm install`)
+  try {
+    mainLogger.log('Doing a log thing')
+    const sortIt = execSync(`. ${app.paths.nvs}/nvs.sh && nvs add 10.11`)
+    mainLogger.log('Doing a log thing part deux')
```
Comment (umaar) [inline installation.js:15]: Can remove?

File: installation.js [hunk 2]
```diff
@@ -2,12 +2,23 @@ const {app} = require('./app.js')
...
+const logger = require('electron-timber')
+const mainLogger = logger.create({name: 'MainBrain'})
 
 const ipc = require('electron-better-ipc')
 
 const installEditorDependencies = (reinstall) => {
   app.notify(`${reinstall ? 'Reinstalling' : 'Installing'} editor dependencies`)
-  execSync(`. ${app.paths.nvs}/nvs.sh && nvs add 10.11 && nvs use 10.11 && cd ${app.paths.editor} && npm install`)
+  try {
+    mainLogger.log('Doing a log thing')
```
Comment (umaar) [inline installation.js:13]: Can remove? Or change the wording?

File: installation.js [hunk 3]
```diff
@@ -60,12 +72,15 @@ const reinstallEditor = async () => {
...
   rimraf.sync(app.paths.editor)
+  mainLogger.log('installing dependencies')
   await installDependencies()
   app.notify('Reinstalled editor', {dismiss: true})
 }
 
 const installDependencies = async () => {
+  mainLogger.log('Installing NVS')
   await installNVS()
+  mainLogger.log('Clonoong editor')
```
Comment (umaar) [inline installation.js:83]: typo

**Review by umaar — APPROVED** [review]
Nice docs 👍  just some feedback on the logging messages

File: installation.js (hunk 3 above)
Comment (solidgoldpig) [reply installation.js:83]: Removed - it was a dupe while debugging

File: installation.js (hunk 2 above)
Comment (solidgoldpig) [reply installation.js:13]: As above

File: installation.js (hunk 1 above)
Comment (solidgoldpig) [reply installation.js:15]: yes - obviously this needs a total and thorough rewrite. But that was just to check whether I was having network issues or nvs had just stopped working

**Pull request merged**